media-video/obs-studio/obs-studio-27.0.1.ebuild

```diff
@@ -0,0 +1,220 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_REMOVE_MODULES_LIST=( FindFreetype )
+LUA_COMPAT=( luajit )
+PYTHON_COMPAT=( python3_{7..9} )
```

Comment:
I did try with Python 10 on May 6th, and it worked for what I tested, but asturm opined it would have been a bold move at the time.  Not that that necessarily meant not to do it, I maybe guess.